lib/requirements.txt

```diff
@@ -1,4 +1,4 @@
-tensorflow

```

Comment:
Can we undo the change on this file which is irrelated to the go microservice?

Reply:
I don't see any changes.